docs/product/explore/profiling/transaction-vs-continuous-profiling.mdx

```diff
@@ -73,3 +75,13 @@ Note, that while the profiling mode can't be changed at runtime, it's fine for d
 ## Differences When Using Sentry
 
 The major difference between continuous profiling and transaction-based profiling when using Sentry, is that with continuous profiling you'll be able to visualize a flamegraph for your entire application. This means, that you'll be able to take a step back from the previous transaction-based view and look at your application's runtime as a whole, which makes it easier to prioritize the functions that are slowing down your entire application and not just one particular transaction.
+
+## Managing Usage
+
+The main difference between transaction and continuous profiling mode is when the profiler is started and stopped. In transaction-based profiling, the profiler is started and stopped based on transactions. In continuous profiling, the profiler is started and stopped by the developer (if ever).
```

Comment:
I think the `(if ever)` part makes this confusing - we could drop that

Reply:
Yes, lets drop it. If ever is confusing